draft-ietf-quic-transport.md

```diff
@@ -1318,9 +1318,10 @@ packet is received from the server, the client MUST use the same value unless it
 abandons the connection attempt and starts a new one. The initial Destination
 Connection ID is used to determine packet protection keys for Initial packets.
 
-A client SHOULD select a Destination Connection ID length long enough to fulfill
-the minimum for every QUIC version it supports. This increases the chance
-subsequent Initial packets are routed to the same server.
+The final version used for a connection might be different from the version of
+the first Initial from the client.  To enable consistent routing through the
+handshake, a client SHOULD select a Destination Connection ID length long enough
```

Comment:
```suggestion
handshake, a client SHOULD select an initial Destination Connection ID length long enough
```

Reply:
Note that accepting this suggestion directly on GitHub will break lint -- you'll need to rewrap.

Reply:
I gave that feedback to github on this feature - that you can handle line lengths.